source/_integrations/rehau_nea_smart_manager.markdown

```diff
@@ -0,0 +1,64 @@
+---
+title: "Rehau Nea Smart Manager"
+description: ""
+ha_release: "0.0.1"
+ha_category: Climate
+ha_iot_class: "Local Polling"
+ha_quality_scale: internal
+ha_config_flow: true
+ha_codeowners:
+  - '@Jeoffreybauvin'
+ha_domain: rehau
+---
+
+Integrates [Rehau Nea Smart Manager](https://www.rehau.com/en-en/nea-smart/francais) Thermostat controller into Home Assistant.
+
+This integration generates a climate entity for each heatarea in your Nea Smart Manager.
+
+To enable this platform, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+climate:
+  - platform: rehau
```

Comment:
We don't allow new platform based integrations. See ADR-0007:

<https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md>

Furthermore, this integration communicates with a device or service, so ADR-0010 also applies:

<https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md>